guides/working-remotely.md

```diff
@@ -0,0 +1,23 @@
+# Working remotely
+
+When a delivery team aren't all in the same place, it's important that we continue to communicate and collaborate as well as we can. This guide contains some suggestions about how to create a happy and productive remote working environment.
+
+## Coming and going
+
+When you arrive, go home, or leave your station for a while, let your teammates know in Slack.
+
+## Know the limits of your technology
+
+Dialing in to a video conference is usually fine for standups, whereas things like all-day workshops are more difficult. If video will be painful, consider attending by speakerphone, or not at all.
+
+## Keep notes
+
+If someone is not able to attend a meeting because they are working remotely, they need to have some kind of report of that meeting in writing. The format doesn't really matter - it could be a set of Trello cards, for instance.
+
+If you need to write out a note, make it as short and sharp as you can.
+
+Be mindful that it is easy for a casual conversation around the office to turn into an ad-hoc meeting. Recognise when this happens, and stop to include remote team members.
+
+## Whiteboards
```

Comment:
Could we add - share document links in advance of any meeting? This is often forgotten and causes frustration when the remote attendee has to ask for them.